src/test/java/com/bobocode/hoverla/bring/testsubject/beandefinition/classbased/ClassBasedBeanDefinitionFieldInjection.java

```diff
@@ -0,0 +1,20 @@
+package com.bobocode.hoverla.bring.testsubject.beandefinition.classbased;
+
+import com.bobocode.hoverla.bring.annotation.Bean;
+import com.bobocode.hoverla.bring.annotation.Inject;
+import com.bobocode.hoverla.bring.annotation.Qualifier;
+import lombok.EqualsAndHashCode;
+import lombok.Setter;
+
+@Bean
+@Setter
+@EqualsAndHashCode
```

Comment:
We don't expect users to implement equals and hashCode for their class based beans

Reply:
fixed